front-end/src/app/reports/f3x/create-workflow/create-f3x-step1.component.ts

```diff
@@ -101,29 +101,68 @@ export class CreateF3XStep1Component implements OnInit, OnDestroy {
     this.f3xSummaryService.getF3xCoverageDates().subscribe((dates) => {
       this.f3xCoverageDatesList = dates;
     });
-    this.form.controls['coverage_from_date'].addValidators(this.buildCoverageDatesValidator('coverage_from_date'));
-    this.form.controls['coverage_through_date'].addValidators(
-      this.buildCoverageDatesValidator('coverage_through_date')
-    );
+    this.form.controls['coverage_from_date'].addValidators([
+        this.buildCoverageDatesValidator('coverage_from_date'),
+        Validators.required
+      ]);
+    this.form.controls['coverage_through_date'].addValidators([
+      this.buildCoverageDatesValidator('coverage_through_date'),
+      Validators.required
+    ]);
 
     // Initialize validation tracking of current JSON schema and form data
     this.validateService.formValidatorSchema = f3xSchema;
     this.validateService.formValidatorForm = this.form;
+
+    //Re-checks the validators on the coverage date fields every second
+    setInterval(()=>{
```

Comment:
I think we really want to avoid running validation on a time schedule.  What prompted this addition?

Reply:
Without this, the validation for a field only runs on form submission and on interaction with the given field.  Normally, this is just fine, but with the edge case this ticket is trying to solve, both fields need to show the validation error because neither one field is at fault.  This is something I would avoid normally, but it seems necessary here, and it comes at a negligible performance cost.  (I ran a performance profile on my machine using the dev panel, and the total runtime of timer events in a 30 second window came out to 180ms)